src/Navbar.js

```diff
@@ -42,3 +159,5 @@ export default CollapsibleNavbar;
 
 
 
+
+
```

Comment:
I'd consider the styling on this and maybe have it inline rather than as a list just to avoid it taking up a lot of screen real estate on mobile when your dropdown menu is open.